packages/server/src/common/operations/operation-action-handler.ts

```diff
@@ -67,8 +67,9 @@ export class OperationActionHandler implements ActionHandler {
         const command = await handler.execute(operation);
         if (command) {
             await this.executeCommand(command);
+            return this.submitModel();
         }
-        return this.modelSubmissionHandler.submitModel('operation');
+        return [];
```

Comment:
This may change the behavior in a way that the GModel is not re-generated after an empty operation. Adopters may have relied on that so this would constitute a breaking change as the previous behavior is also valid and may not strictly be classified as a bug.

Reply:
IMO this is more of a bugfix to ensure that the operation handler works as expected.
Nevertheless your are right in certain edge cases adopters may have relied on a model update
(I can`t think of any particular usecase where a effectivly no-op update would be required but who knows).
Maybe we should at least refactor this in overrideable method or property to configure whether update on empty commands should be enabled?
We disable it by default but atleast it gives adopters an easy entry point to change the behavior.

Reply:
I agree now that we can consider this more of a bugfix since the [Java server was behaving like this all along](https://github.com/eclipse-glsp/glsp-server/blob/8604c5d1997cf9a743a4fe586f43ff0aa3ee1c0c/plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/operations/OperationActionHandler.java#L62-L69). So I would almost argue that we do not need an additional flag. The method as-is is already very short and could be overwritten, don't you think?

Reply:
👍🏼 Fine with me.